Meta/Lagom/Libraries/LibGUI/SDLServer.cpp

```diff
@@ -38,21 +38,20 @@ SDLServer::SDLServer()
             if (event.window.windowID == 0)
                 continue;
 
+            u32 buttons = 0;
```

Comment:
This seems like it was a bug with the first Lagom GUI commit - if so it should be squashed into that commit

Reply:
Yes it was, but I’d like to give credit for the discovery.

Reply:
I see, I missed the co-author line, fair enough